flink_table.go

```diff
@@ -0,0 +1,116 @@
+package aiven
+
+type (
+	// FlinkTableHandler aiven go-client handler for Flink Jobs
+	FlinkTableHandler struct {
+		client *Client
+	}
+
+	// CreateFlinkTableRequest Aiven API request
+	// POST https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/table
+	CreateFlinkTableRequest struct {
+		IntegrationId string `json:"integration_id"`
+		JDBCTable     string `json:"jdbc_table,omitempty"`
+		KafkaTopic    string `json:"kafka_topic,omitempty"`
+		LikeOptions   string `json:"like_options,omitempty"`
+		Name          string `json:"name"`
+		PartitionedBy string `json:"partitioned_by,omitempty"`
+		SchemaSQL     string `json:"schema_sql"`
+	}
+
+	// CreateFlinkTableResponse Aiven API response
+	// POST https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/table
+	CreateFlinkTableResponse struct {
+		APIResponse
+
+		flinkTable
+	}
+
+	// GetFlinkTableResponse Aiven API request
+	// GET https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/table/<table_id>
+	GetFlinkTableRequest struct {
+		TableId string `json:"table_id"`
+	}
+
+	// GetFlinkTableResponse Aiven API response
+	// GET https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/table/<table_id>
+	GetFlinkTableResponse struct {
+		APIResponse
+
+		flinkTable
+	}
+
+	// DeleteFlinkTableResponse Aiven API response
```

Comment:
Should be a `DeleteFlinkTableRequest` here I think

Reply:
:heavy_check_mark: